lib/evss/documents_service.rb

```diff
@@ -1,26 +1,62 @@
 # frozen_string_literal: true
+require 'common/exceptions'
 require_dependency 'evss/base_service'
 
 module EVSS
+  # DocumentData is a struct class bundling together the required attributes of a file upload
+  DocumentData = Struct.new(
```

Comment:
I thought about that. It's weird that we have some of our stuff in /lib/ with an EVSS namespace but not all of it. When it was a struct, the document was small enough to just leave with the client, but it's probably too big now that it's doing double duty as a validator.